### PR TITLE
Add importAll command

### PR DESCRIPTION
**Pull request by yuhongherald**

### CODE REVIEW

**Comment by yuhongherald** [comment]
@eugenepeh you can focus on looking in the session package for:
- how the classes are documented and whether they are appropriate.
- SLAP, delimiter violation

File: src/main/java/seedu/address/logic/commands/CommandWords.java [hunk 1]
```diff
@@ -152,6 +144,22 @@ public void setCommandWord(String currentWord, String newWord) throws CommandWor
         throw new CommandWordException(getMessageUnused(currentWord));
     }
 
+    /**
+     * throws a (@code CommandWordException) if (@code currentWord) or (@code newWord) is not valid
```
Comment (eugenepeh) [inline src/main/java/seedu/address/logic/commands/CommandWords.java:148]: Method headers need to be more specific. 
E.g. given a random developer, will they understand what your method is doing (without looking into implementation) by looking at header comments?

File: src/main/java/seedu/address/logic/parser/ImportAllCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import static seedu.address.logic.parser.ParserUtil.parseFilename;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.commands.ImportAllCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+//@@author yuhongherald
+
+/**
+ * Parses input arguments and creates a new AddCommand object
```
Comment (eugenepeh) [inline src/main/java/seedu/address/logic/parser/ImportAllCommandParser.java:13]: AddCommand?

File: src/main/java/seedu/address/model/session/JobEntry.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+import seedu.address.model.person.Person;
+import seedu.address.model.person.UniqueEmployeeList;
+import seedu.address.model.remark.Remark;
+import seedu.address.model.remark.RemarkList;
+
+//@@author yuhongherald
+/**
+ * Represents a job entry in an (@link ImportSession)
+ */
+public class JobEntry extends Job implements ExcelRowReference {
```
Comment (eugenepeh) [inline src/main/java/seedu/address/model/session/JobEntry.java:23]: Why is the `ExcelRowReference` necessary? Doesnt it belongs to storage?

**Review by eugenepeh — COMMENTED** [review]
It seems to be too large for a PR, you need to start updating the documentation and diagrams, otherwise it doesn't provide other developers the knowledge of how all this fit together.

**Pull request merged**